app/src/main/res/menu/drawer_view.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="utf-8"?>
+<menu xmlns:android="http://schemas.android.com/apk/res/android">
+
+    <group android:checkableBehavior="single" android:id="@+id/navigation_menu">
+
+        <item
```

Comment:
Is this XML file used? I couldn't see it anywhere. I think the menu is setup dynamically.